tests/intern.js

```diff
@@ -10,7 +10,9 @@ define([
   'intern/node_modules/dojo/has!host-node?intern/node_modules/dojo/topic',
   './tools/firefox_profile'
 ],
-function (intern, topic, firefoxProfile) {
+function (require, intern, topic, firefoxProfile) {
+  var Duration = require('durationJs');
```

Comment:
@larissagaulia undo the changes in `tests/intern.js` and `tests/server/*` just like `tests/functional/*`